dm/templates/logsink/logsink.py

```diff
@@ -19,6 +19,7 @@ def create_pubsub(context, logsink_name):
 
     properties = context.properties
     project_id = properties.get('project', context.env['project'])
```

Comment:
Instead of the 2 line to set and overwrite project_id, I suggest a single line:
```suggestion
    project_id = properties.get('destinationProject', properties.get('project', context.env['project']))
```

Please update all places